### PR TITLE
[READY] Plasmaman sprite changes.

### PR DESCRIPTION
**Pull request by pireamaineach**
## About The Pull Request

Having not bad sprites is good.
newsuits

## Why It's Good For The Game

Having not bad sprites ? Pretty good.
Pseudo-changelog for unnecessary detail:
Reprites the prisoner and EMT envirosuits, the later to be more consistent with other med-bay suits, and the former to bring it in line with other changes in the P.R., and also make it look less like total shit.
Brings the stock envirosuit in line with the general departmental sprite design.
Fixes that weird space where the stripes on the helmet icons disappear into nothing.
Fixes the weird colour-changing eyes on the welding visor for helmets.
Fixes the icon for the default helmet which was at some point replaced with the on mob for some reason.
Also adds the lit prisoner helmet because I guess they didn't think that was important enough to spend 2 minutes making.

Also adds gloves now which solve the clipping issues of plasmamen.
Functionally they are the same as black gloves.
Also fixes the hand sprite clipping through the body of the suit.
newsuitsbutactually

## Changelog
:cl:
imageadd: Resprites several plasmaman items. 
add:Adds plasmagloves and fixes clipping on the plasmaman suit.
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
Throw some good comparison shots in the PR

**Comment by pireamaineach** [comment]
> Throw some good comparison shots in the PR

We have an icon difference bot.

**Comment by RaveRadbury** [comment]
Would you like me to reach out to you when I've got anything incoming that will require a new plasmaman suit?

**Comment by pireamaineach** [comment]
> Would you like me to reach out to you when I've got anything incoming that will require a new plasmaman suit?

Sure, I've been on hiatus but I'm good to work on anything involving plasmamen.

**Comment by ATH1909** [comment]
> > Throw some good comparison shots in the PR
> 
> We have an icon difference bot.

but that requires effort and time for us plebians to use

**Comment by pireamaineach** [comment]
> > > Throw some good comparison shots in the PR
> > 
> > 
> > We have an icon difference bot.
> 
> but that requires effort and time for us plebians to use

It's in checks. Click checks and look at the images.

**Comment by Exavere** [comment]
Huuuuuurngh, yes.

**Comment by spookydonut** [comment]
> 
> 
> > Throw some good comparison shots in the PR
> 
> We have an icon difference bot.

Doesn't show how they look on the actual mobs

**Comment by SouDescolado** [comment]
Are you only messing with sprites or are you also messing with code?

In case its the second option, you could check https://github.com/tgstation/tgstation/blob/master/code/game/machinery/gulag_teleporter.dm so it changes the plasmaman suit automatically

**Comment by pireamaineach** [comment]
> Are you only messing with sprites or are you also messing with code?
> 
> In case its the second option, you could check https://github.com/tgstation/tgstation/blob/master/code/game/machinery/gulag_teleporter.dm so it changes the plasmaman suit automatically

That code simply has a list of items that are needed to sustain life and doesn't take them. The prisoner envirosuits are for roundstart prisoners.

**Comment by pireamaineach** [comment]
I'll get up some images later today.

**Comment by ShizCalev** [comment]
pics please

**Comment by pireamaineach** [comment]
> pics please

Dream Daemon refuses to work

**Comment by pireamaineach** [comment]
Whoopsies, I fix mime file

**Comment by pireamaineach** [comment]
Why Icon bot broke

**Comment by Arturlang** [comment]
@pireamaineach  @AffectedArc07, aka the guy who hosts it, was having some problems with the diffbot not working so he migrated it, should work now though

**Comment by pireamaineach** [comment]
@ShizCalev Can you force the checks to retry ? I was told this was a thing.

**Comment by AffectedArc07** [comment]
Travis can be re-run, but I am unsure if IDB can. I migrated IDB to a newer, more powerful machine, but I forgot to check dependencies since I was in a hurry to get it back online. If you want to force-kick it into working, just make a commit.

**Comment by pireamaineach** [comment]
Spritebot me angry

**Comment by AffectedArc07** [comment]
Try again, it was having some issues, should be working now

**Comment by pireamaineach** [comment]
Removed the new name because the more names we add the less room there is for creativity without just making shit up.

**Comment by pireamaineach** [comment]
@AffectedArc07 Thanks, I'm going to give you a big sloppy kiss. 
Nice bot.

**Comment by pireamaineach** [comment]
Fuck you Travis FUCK

**Comment by pireamaineach** [comment]
Why can't these bots just fucking WORK FUCK WHY

**Comment by pireamaineach** [comment]
Okay, Dream Daemon finally working. I will throw together quick pictures.

**Comment by pireamaineach** [comment]
Probably just the new EMT and Prisoner suits, short on time.

**Comment by pireamaineach** [comment]
Picture added

**Comment by pireamaineach** [comment]
@ShizCalev

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -65,3 +65,71 @@
...
+
+/obj/item/clothing/gloves/color/plasmaman/prototype
+	name = "prototype envirogloves"
+	icon_state = "protoplasma"
+	item_state = "protoplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/clown
+	name = "clown envirogloves"
+	icon_state = "clownplasma"
+	item_state = "clownplasma"
```
Comment (trollbreeder) [inline code/modules/clothing/gloves/miscellaneous.dm]: Your linters, hand them over.

Comment (pireamaineach) [reply code/modules/clothing/gloves/miscellaneous.dm]: what is happen

Comment (pireamaineach) [reply code/modules/clothing/gloves/miscellaneous.dm]: explain

Comment (trollbreeder) [reply code/modules/clothing/gloves/miscellaneous.dm]: I looked at the files changed and saw the pesky no newline icon, so i thought, hey, let's review this so pire notices.

Comment (trollbreeder) [reply code/modules/clothing/gloves/miscellaneous.dm]: Resolved due to commit 57a4997

Comment (pireamaineach) [reply code/modules/clothing/gloves/miscellaneous.dm]: this is very good and epic I'm going to give you a big sloppy kiss :flushed:

Comment (pireamaineach) [reply code/modules/clothing/gloves/miscellaneous.dm]: how come spritebot is kill

**Comment by pireamaineach** [comment]
How come spritebot is kill

**Review by kriskog — APPROVED** [review]

**Pull request merged**